server/src/main/java/org/opensearch/index/mapper/MapperService.java

```diff
@@ -162,7 +162,19 @@ public enum MergeReason {
     // Deprecated set of meta-fields, for checking if a field is meta, use an instance method isMetadataField instead
     @Deprecated
     public static final Set<String> META_FIELDS_BEFORE_7DOT8 = Collections.unmodifiableSet(
-        new HashSet<>(Arrays.asList("_id", IgnoredFieldMapper.NAME, "_index", "_routing", "_size", "_timestamp", "_ttl", "_type"))
```

Comment:
As `NestedPathFieldMapper.NAME` is introduced in `2.0`; do we need this as meta field in < 7.8 version ?

Reply:
Alas, no! 2.0 is only bwc to 7.10.0 indexes. Thx!